### DISCUSSION TITLE
Multilingual Chants List - لیست چندزبانه شعارها

### DISCUSSION THREAD

**Discussion by yasaman72** [WeAreMahsaAmini/Core#387] [Ideas]
Create a list of all chants in multiple languages. People can suggest and translate chants. It can contain a sample audio showing how it should be said.

لیستی بسازیم از تمام شعارهای این روزها به زبان‌های مختلف. مردم بتوانند به لیست شعارهای بیشتر اضافه کنند یا ترجمه شعارها را بفرستند. علاوه بر متن شعارها فایل صوتی که نحوه‌ی خوانده شدن شعار رو نشون می‌ده داشته باشیم.

**Comment by karimifar** [comment]
Love this. It is much needed.

**Comment by theStratus** [comment]
brilliant idea
could you please make a repository on that? so that people can contribute

**Comment by yasaman72** [comment]
I've made a Google Sheet for chants. It's WIP:
https://docs.google.com/spreadsheets/d/1w0H2AjCpgUHJzWlpVDr5vs4iGb22UWMbWlVNCjMTz50/edit?usp=drivesdk

**Reply by theStratus** [reply]
I made a repository on that. thank you. 
https://github.com/theStratus/SlogansMahsaAmini
